notebooks/3_KernelMethods.ipynb

```diff
@@ -81,7 +81,30 @@
     "    k(\\mathbf{x}, \\mathbf{x}^{\\prime}) = \\exp{\\left(-\\gamma\\lVert \\mathbf{x} - \\mathbf{x}^{\\prime}\\rVert^2\\right)}.\n",
     "\\end{equation}\n",
     "\n",
-    "In this notebook we use the gaussian kernel (```gaussian_kernel(XA, XB, gamma=1.0)```), however we have also included the linear kernel function (```linear_kernel(XA, XB)```) in ```utilities/kernels.py```. You can check rather easily that if you use a linear kernel all of the kernel methods reduce explicitly to linear regression or principal component analysis in the original feature space."
+    "If the Gaussian kernel is used, we implicitly define an infinite dimensional feature space. To see this in detail, let's consider the simple case that $\\gamma = 1$ and  $ \\mathbf{x}, \\mathbf{x}^{\\prime}$ are scalars, which means each sample has only one feature.\n",
```

Comment:
Where did you get this example? I'm very doubtful that it will be helpful at this section of the notebook.

Reply:
If you would like to include it, at minimum it needs to be re-written to be clearer.

Reply:
I got it from the ML lecture notes. It does help me understand why the RKHS space may be infinite-dimensional. Do you have any suggestions on how to improve it? I think it's clear enough.
![image](https://user-images.githubusercontent.com/56018055/98259175-109f0a80-1f82-11eb-8112-059f0186d76c.png)
![image](https://user-images.githubusercontent.com/56018055/98260480-ae470980-1f83-11eb-8dd2-bd16ed13e829.png)